### ISSUE TITLE
Unusual use needed for me

### ISSUE THREAD

**Issue by dbessa1** [transistorsoft/cordova-background-geolocation-lt#661]
Is there a problem sending the #Start command before sending the first #Configure?

In this case would it be necessary to send a #stop followed by #Start so that the instructions from the first configure actually apply?

(I'm using ios)

**Comment by dbessa1** [comment]
I am using version 2.8.4 before the #ready command exists

**Comment by christocracy** [comment]
>Is there a problem sending the #Start command before sending the first #Configure?

Yes there's a problem.  **Don't**.

You **MUST** execute `#configure` **BEFORE** you `#start`.

**Comment by dbessa1** [comment]
Is it possible for me to apply an emergency fix by replacing #configure with #Setconfig?

**Comment by christocracy** [comment]
I have no idea what you're trying to do.  The first thing you do when `deviceready` fires is listen to events and `#configure` the plugin.

Use `#setConfig` only after the callback to `#configure` fires.

```javascript
BackgroundGeolocation.configure(myConfig, function(state) {
  BackgroundGeolocation.setConfig(myNewConfig);
});
```

**Comment by dbessa1** [comment]
I did this to ensure the #start of the plugin, because my app connects to the internet first to download all its code, including those of gps. I did so to be able to update many codes without having to resend to the store.
If the device has no internet signal at the time of the hidden opening of the app (by some event geofence); I was afraid the system will shut down the app before the internet signal comes back. Being the plugin in #start, the plugin would guarantee a quick reopening in this case, since I configured the plugin to spend most of the time off just listening to geofences.
